pkg/reduce/reduce.go

```diff
@@ -51,13 +53,15 @@ type DataForward struct {
 
 func NewDataForward(ctx context.Context,
 	udf applier.ReduceApplier,
+	vertex *dfv1.Vertex,
```

Comment:
can we be more explicit about what we are passing to the constructor instead of passing the Vertex object?

Reply:
I was referring to https://github.com/numaproj/numaflow/blob/main/pkg/isb/forward/forward.go#L63 I added `vertexName` and `pipelineName` to `DataForward` such that we can use them as labels when emitting reducer metrics.